integration/out_test.go

```diff
@@ -17,6 +17,7 @@ import (
 
 	"github.com/frodenas/gcs-resource"
 	"github.com/frodenas/gcs-resource/out"
+	"github.com/mholt/archiver"
```

Comment:
This introduces some dependencies, and they are not vendored. We will need to add them, but it can be after the PR is merged.